README.md

```diff
@@ -13,9 +30,20 @@ Learning assembly for linux-x64
   * [Say hello to x86_64 Assembly part 7](https://0xax.github.io/asm_7/)
   * [Say hello to x86_64 Assembly part 8](https://0xax.github.io/asm_8/)
 
+## Requirements
+
+To run code examples you will need following tools:
+
+- [64-bit distribution of Linux](https://en.wikipedia.org/wiki/Linux_distribution)
+- [make](https://www.gnu.org/software/make/)
+- [nasm](https://nasm.us/)
```

Comment:
this should be fixed to NASM

Reply:
Fixed in the [7d399c3](https://github.com/0xAX/asm/pull/20/commits/7d399c3098ad08968d739026365edc3a9fcaa905)